doc/modules/changes/20250616_ranpengli

```diff
@@ -0,0 +1,7 @@
+Added: The entropy model now supports multiple components.
+
+The method assumes there is a background field, whose percentage doesn't need to be specified.
+
+The adiabatic reference profile is calculated based on the background entropy.
```

Comment:
remove empty lines in changelog entries, they would look awkward in the release log.